### ISSUE TITLE
1.8.9 compatibility?

### ISSUE THREAD

**Issue by Kittenhugger213** [gri573/rethinking-voxels#260]
I would like to request 1.8.9 compatibility if you would be willing to do that.

**Comment by Kittenhugger213** [comment]
Could you clarify if that would include 1.8.9?

**Comment by KaseyTheJackal** [comment]
Optifine isn't supported beyond Beta 18d

**Comment by Kittenhugger213** [comment]
![2023-11-27_20 39 41](https://github.com/gri573/rethinking-voxels/assets/144856507/c35c2884-ae42-4989-85e6-d1469a535b31)
this happens when I use beta 18d on 1.8.9, know any fixes?

**Comment by Kittenhugger213** [comment]
It shifts around when I move and keeps the same position relative to me.
![2023-11-27_20 49 05](https://github.com/gri573/rethinking-voxels/assets/144856507/70bec4b9-e6d9-422d-80eb-372b81640e3e)
![2023-11-27_20 49 10](https://github.com/gri573/rethinking-voxels/assets/144856507/e2f9da7d-0281-47b7-ad3b-4ef81ad4c742)

**Comment by Kittenhugger213** [comment]
if it helps my graphics card is an intel hd graphics 4600.

**Comment by Kittenhugger213** [comment]
![2023-11-27_21 00 23](https://github.com/gri573/rethinking-voxels/assets/144856507/b4084c13-471d-45da-b19d-53833d1dabb8)

**Comment by KaseyTheJackal** [comment]
> if it helps my graphics card is an intel hd graphics 4600.

Yeah you're never gonna get anything acceptable with that. I run an RTX 3070Ti laptop GPU and I struggle
